### PR TITLE
PR: Use justified text instead of default left-alignment

### LINKED ISSUE #170: Image centering sometimes looks off
## Issue Description

The images don't look centered since the text is left aligned as discussed with @ccordoba12 

### Where is the relevant portion of the documentation located?

Images in the docs
image

### What would you suggest it be replaced with?

Check a way to improve the centering of the images (maybe adding some negative margin to the right of paragraphs to enable them covert more width even if they are left-aligned).

### Other information

* Is the problem specific to documentation for a particular version of Spyder?

* Are you planning on submitting a Github pull request with a suitable change?

* Anything else you'd like to add?

### PR DESCRIPTION
**Pull request by dalthviz**
### Pull Request Checklist

* [x] Read and followed this repo's [Contributing Guidelines](https://github.com/spyder-ide/spyder/blob/master/CONTRIBUTING.md)
* [x] Based your PR on the latest version of the correct branch (master or 3.x)
* [x] Checked your writing carefully for correct English spelling, grammar, etc
* [x] Described your changes and the motivation for them below
* [x] Noted what issue(s) this pull request resolves, creating one if needed

## Description of Changes

Centered images look off with left-aligned text

### Issue(s) Resolved

Fixes #170

### CODE REVIEW

**Comment by juanis2112** [comment]
Yes @ccordoba12 , I do remember that's what we agreed on. However, @dalthviz can you post a screenshot to preview this change you made to see how it looks? Maybe this works too.

**Comment by dalthviz** [comment]
@ccordoba12 I was thinking in the option of changing the margin and at the end I decided to not go through that path since it could lead to cases where the images keep looking off and also it will mess up with the responsiveness of the page. @juanis2112 you can use the Netlify preview link to check how things are looking (you can see it if you click `Show all checks`)

**Comment by ccordoba12** [comment]
>  However, @dalthviz can you post a screenshot to preview this change you made to see how it looks? Maybe this works too.

@juanis2112, we have Netlify for that now. By pressing the `Details` link shown below, you'll get a live preview of this PR:

Selección_025

So you can browse all content and see how it looks with the text justified.

**Comment by ccordoba12** [comment]
> I decided to not go through that path since it could lead to cases where the images keep looking off and also it will mess up with the responsiveness of the page.

Good point! I hadn't consider it!

Then I think we should go with the justified text.

**Comment by juanis2112** [comment]
> > However, @dalthviz can you post a screenshot to preview this change you made to see how it looks? Maybe this works too.
> 
> @juanis2112, we have Netlify for that now. By pressing the `Details` link shown below, you'll get a live preview of this PR:
> 
> Selección_025
> 
> So you can browse all content and see how it looks with the text justified.

Sorry I completely forgot about this. I'll try it now

**Comment by juanis2112** [comment]
I think justified text is fine, except for the text in the cards. I think we should leave that as it was before (centered) because otherwise looks weird. What do you think @ccordoba12 and @dalthviz ?

**Comment by ccordoba12** [comment]
> I think justified text is fine, except for the text in the cards.

I was thinking exactly the same @juanis2112!

@dalthviz, is it possible to change that in `custom_styles.css`? Or does it need to be changed in the Sphinx theme?

File: doc/_static/custom_styles.css [hunk 1]
```diff
@@ -1,3 +1,8 @@
+/* Correctly wrap paragraphs*/
+p {
+  text-align: justify;
+}
+
```
Comment (CAM-Gerlach) [inline doc/_static/custom_styles.css:5]:
```suggestion

.card-text {
    text-align: center;
}

```

This will keep the card text centered.

Comment (juanis2112) [reply doc/_static/custom_styles.css:5]: I think it works here @ccordoba12. If you agree with @CAM-Gerlach suggestion, @dalthviz, can you commit it please?

**Comment by juanis2112** [comment]
Should we close this PR as we will not justify text @dalthviz, @ccordoba12  ?

**Comment by ccordoba12** [comment]
Agreed. Let's close it.

**Pull request closed without merging**